packages/migrate/src/__tests__/handlePanic.migrate.test.ts

```diff
@@ -176,13 +176,16 @@ describe('handlePanic migrate', () => {
         draft: false,
         prismaSchema: migrate.getPrismaSchema(),
       })
-    } catch (error) {
+    } catch (e) {
+      const error = e as RustPanic
+
       expect(error).toMatchSnapshot()
       expect(JSON.parse(JSON.stringify(error))).toMatchObject({
         area: 'LIFT_CLI',
         schemaPath,
       })
-      expect(error.rustStack).toContain('This is the debugPanic artificial panic')
+      expect(error.message).toContain('This is the debugPanic artificial panic')
+      expect(error.rustStack).toBeTruthy()
```

Comment:
Can we snapshot this as well?

Reply:
That would yield different results on every OS, I can perhaps snapshot a subset containing `std::panicking` or something like that